rdr_service/resource/generators/participant.py

```diff
@@ -1892,52 +1893,68 @@ def get_consent_pdf_validation_status(p_id, consent_response_rec,
         pdf_validation_start_date = datetime.datetime(2023, 3, 11, 1, 34, 2)
         status = BQModuleStatusEnum.SUBMITTED  # Default, this method is only called for "Yes" consents that have PDF
         dao = None
-        if not ro_session:
-            dao = ResourceDataDao(backup=True)
-            ro_session = dao.session()
-
         # TODO:  If SUBMITTED_NOT_VALIDATED status is expanded to use for other non-EHR consents:
         # Reminder that Primary and CABOR consent types in the consent_file table have their own validation records,
         # but share the same questionnaire_response_id.  May need to add a match on consent_file.type
-        if module_name != 'EHRConsentPII':
-            logging.warning(f'PDF Validation status for consent {module_name} not currently supported')
-            return status
-
-        qr_ids = []
-        if consent_response_rec.answerHash:
-            # PDR-1795: Need to look for duplicates of this response to find relevant consent validation result
-            response_id_sql = f"""
-                select questionnaire_response_id from questionnaire_response
+        if module_name == 'EHRConsentPII':
+            if not ro_session:
+                dao = ResourceDataDao(backup=True)
+                ro_session = dao.session()
+
+            # Check for RDR overrides to SUBMITTED_NOT_VALIDATED/SUBMITTED_INVALID, which may have been manually
+            # applied to participant_summary to allow participants to continue to PM&B.   If the consent_response
+            # being checked matches what's in participant_summary on authored date, use participant_summary status
+            ps_sql = f"""
```

Comment:
FYI:  The code has been reorganized so the diff appears more extensive than it is.  The main change is inserting this new `ps_sql` query and potential early return if this check determines we should use the `consentForElectronicHealthRecords` status value from the `participant_summary` record, instead of forcing a match on a  `consent_file` validation record that indicates successful validation.